### PR TITLE
Adresses #3215 NTRs ureter

### PR DESCRIPTION
**Pull request by aleixpuigb**
Adresses #3215 NTRs ureter.
Thir PR includes the three segments of the ureter, and subclasses for right and left for each segment.

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224531,6 +224531,148 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600108
+name: proximal ureter
+def: "A segment of an ureter that extends from the renal pelvis to the common iliac artery bifurcation (to form the internal and external iliac arteries)." [PMID:30422575]
```
Comment (Caroline-99) [inline src/ontology/uberon-edit.obo:224537]: A segment of **the** ureter instead of **an**

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224531,6 +224531,148 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
...
+relationship: dc-contributor https://orcid.org/0000-0001-7655-4833 ! Ellen Quardokus
+relationship: dc-contributor https://orcid.org/0000-0002-4341-3265
+relationship: dc-contributor https://orcid.org/0000-0003-2804-127X ! Sanjay Jain
+relationship: part_of UBERON:0000056 ! ureter
+property_value: dcterms-date "2024-08-28T14:15:03Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600110
+name: intramural ureter
+def: "The terminal segment of the ureter that is continuous with the distal ureter and lies where the ureter enters the bladder. The intramural ureter is aperistaltic, serving as a reflux-preventing mechanism, preventing the backflow of urine from the bladder into the ureters." [PMID:14142154, PMID:30422575]
```
Comment (Caroline-99) [inline src/ontology/uberon-edit.obo:224570]: I suggest making it clear that intramural ureter is located within the urinary bladder wall ie. that lies where the ureter enters the bladder and extends within the bladder wall.

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224531,6 +224531,148 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
...
+synonym: "intravesical ureter" EXACT [doi:10.53347/rID-23183]
```
Comment (Caroline-99) [inline src/ontology/uberon-edit.obo:224571]: Found this reference as well PMID: [33085409](https://pubmed.ncbi.nlm.nih.gov/33085409)

**Review by Caroline-99 — CHANGES_REQUESTED** [review]

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224537]: It's true I haven't been consistent, I will fix it, thank you!

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Review by Caroline-99 — APPROVED** [review]

**Pull request merged**